apps/src/experimentActionCableLoad/experiment-actioncable-load.js

```diff
@@ -25,13 +33,21 @@ const testLoad = function () {
         logEvent('ActionCableLoadTestingConnected', connectionId);
 
         setTimeout(() => {
-          channel.echo(connectionId);
+          if (channel) {
```

Comment:
Doesn't seem harmful + who cares in throwaway code, but curiosity: is there a reason you're guard checking against channel not existing?